py/ai/h2o/sparkling/Initializer.py

```diff
@@ -141,3 +147,13 @@ def __add_url_to_classloader(jvm, url):
             else:
                 logger.debug("Skipping classloader '{}'".format(cl.toString()))
             cl = cl.getParent()
+
+    @staticmethod
+    def getVersion():
+        here = path.abspath(path.dirname(__file__))
+        if '.zip' in here:
+            with zipfile.ZipFile(here[:-len("ai/h2o/sparkling/")], 'r') as archive:
+                return archive.read('ai/h2o/sparkling/version.txt').decode('utf-8').strip()
```

Comment:
Can happen that the version file is still in the `pysparkling` folder?

Reply:
Good question, but not anymore, we are generating it into `ai/h2o/sparkling`